### ISSUE TITLE
File Not Found Messaging Inconsistent

### ISSUE THREAD

**Issue by dkelosky** [zowe/zowe-cli#95]
`zowe files upload ftds local remote` gives a message like:
```
Command Error:
Node.js File System API error
```

`zowe jobs submit lf noexist`
```
Unexpected Command Error:
Please review the message and stack below.
Contact the creator of handler:
"C:\Users\xxx\AppData\Roaming\npm\node_modules\@brightside\core\lib\zosjobs\
src\cli\submit\local-file/../Submit.shared.handler"
Message:
ENOENT: no such file or directory, open 'C:\Users\xxx\Work\Dev\node\doesnot\
exist'
Stack:
Error: ENOENT: no such file or directory, open 'C:\Users\xxx\Work\Dev\node\d
```

It seems one command uses `fs.readFileSync` directly and another uses a custom encapsulation of `fs.readFileSync`.  It seems they should use the same encapsulation for consistent messaging, perhaps `IO.readFileSync`?

**Comment by AHumanFromCA** [comment]
Do we already have an IO.readFileSync in place for zowe-cli?

**Comment by dkelosky** [comment]
It looks to be in imperative

**Comment by MarkAckert** [comment]
Confirmed this still exists. The error message from the first command does show ENOENT so users can figure out what happened, but the formatting between the 2 command errors is different.

**Comment by ATorrise** [comment]
This issue has been addressed with #2078
